jest.config.cjs

```diff
@@ -51,12 +51,12 @@ const customJestConfig = {
   coverageProvider: "v8",
 
   // A list of reporter names that Jest uses when writing coverage reports
-  // coverageReporters: [
-  //   "json",
-  //   "text",
-  //   "lcov",
-  //   "clover"
-  // ],
+  coverageReporters: [
+    // "json",
+    "text",
+    // "lcov",
```

Comment:
I think this is the one that generates the HTML report that I use to check if there's any code I forgot to write tests for:

```suggestion
     "lcov",
```